frontend/src/constants/API.js

```diff
@@ -38,3 +38,12 @@ export const REMOVE_MINE_EXPECTED_DOCUMENT = (expectedDocumentGuid, mineDocument
 export const MINE_MANAGER_HISTORY = (mineNo) =>
   `/parties/mines/manager-history/csv?mine_no=${mineNo}`;
 export const MINESPACE_USER = "/users/minespace";
+
+// permits
+export const PERMIT = (params) =>
+  params ? `/permits?${queryString.stringify(params)}` : "/permits";
```

Comment:
Setting `params` to a default value of an empty object should mean that you can just pass it in directly without the ternary

Reply:
True. Modified.

Reply:
Did that change not get pushed?

Reply:
I reverted it. I think the API is behaving differently if the question mark is in the call. I can pursue this but would prefer to do it in our uploading permit documents branch where we are making other API changes.

Reply:
Ahh gotcha. Not a big deal. Marking this as resolved